### DISCUSSION TITLE
舌像检测分割数据集征集

### DISCUSSION THREAD

**Discussion by AyangngY** [linhandev/dataset#13] [数据集收集]
请填写数据集基本信息，*为必填项，其他内容请尽量填写．

注：开 Issue 后数据集成功并入项目，您的Github ID不会作为项目合作者在项目主页展示，我们会在项目末尾致谢．如果您希望作为项目合作者被展示请[提交PR](https://github.com/linhandev/dataset/pulls)．

名称*：
标注内容*（标注的器官/病灶）：	
LICENSE*（关于数据集使用和传播的协议，常用LISENCE写名字，如果是Custome LICENSE写网址）：
下载方式*（下载网址或介绍下载方式的网页）：
类型（分类/分割/检测/...）：	
模态（CT/MR/CXR/...）：	
数量（扫描总数/图片张数）：	
标签格式：	
文件格式：

**Comment by linhandev** [comment]
感谢大佬issue，请填写数据集基本信息🤔

**Comment by linhandev** [comment]
如果有大佬研究过这个方向，欢迎回帖相关的数据集，我们会在项目末尾致谢！
